docs/docs/pricing/README.md

```diff
@@ -131,9 +128,8 @@ This would result 0 credits per run because:
 
 Consider an a source (like **RSS - New Item in Feed** for instance) that takes 60 seconds total to finish polling, per execution.
 
-Each execution of this source would result 1 credit because:
+Each execution of this source would result 0 credits because:
 
-- The first {{ $site.themeConfig.base_credits_price.seconds }} of computation time per source execution is included.
 - The **RSS - New Item in Feed** source is a [publicly available component](https://pipedream.com/apps/rss/triggers/new-item-in-feed).
```

Comment:
Since there's only one bullet here, just combine this with the sentence above, i.e.

> Each execution of this source would result in 0 credits because the **RSS - New Item in Feed** source is a [publicly available component](https://pipedream.com/apps/rss/triggers/new-item-in-feed).